From 3bdc30e8bf3e10e6b6b283d216806d8e5d333c1a Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 4 Mar 2019 14:13:27 -0500
Subject: [PATCH] TOOLS-2193 TOP Makefile.defs assignment complicates
 overriding Reviewed by: Tim Foster <tim.foster@joyent.com> Reviewed by: Trent
 Mick <trentm@gmail.com> Approved by: Tim Foster <tim.foster@joyent.com>

---
 tools/mk/Makefile.defs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/mk/Makefile.defs b/tools/mk/Makefile.defs
index fa3569e..8bd6ac9 100644
--- a/tools/mk/Makefile.defs
+++ b/tools/mk/Makefile.defs
@@ -53,7 +53,7 @@
 # destinations, but sometimes to modify the behavior of the build.
 #
 
-TOP := $(shell pwd)
+TOP ?= $(shell pwd)
 
 #
 # Mountain Gorilla-spec'd versioning.
-- 
2.21.0

